### DISCUSSION TITLE
Performance Hook GC Kind/Flag documentation

### DISCUSSION THREAD

**Discussion by JMTK** [nodejs/help#4659] [Q&A]
Where can I find more information about the GC Flags and Kind found here: https://nodejs.org/api/perf_hooks.html#performancenodeentrydetail

I started logging GCs using Performance Observer and have been interested in learning more about the differences in the Flags, and also the differences between Major/Minor/Incremental. 

I tried googling it to see if they were v8 flags but I didn't find anything. 

Thanks!

**Comment by DarkGL** [comment]
https://github.com/nodejs/node/blob/main/src/node_perf.h
